docs/UserGuide.md

```diff
@@ -614,7 +614,7 @@ RC4HDB saves your data after every command. There is no need to save manually.
 * All file commands have a `FOLDER_NAME` or `FILE_NAME` field depending on the command that is being used. 
 * `FOLDER_NAME` signifies the name of the **sub** data folder that is being used.
 * `FILE_NAME` refers to the [CSV](#glossary-of-terms) file to be targeted.
-* Do not include any file types in the `FOLDER_NAME` or `FILE_NAME`.
+* Do not include any file types in the `FOLDER_NAME` or `FILE_NAME`. A file type is the description that comes after the`.` in a file. For example, The file name of the file `residents.csv` is  just `residents`.
```

Comment:
Minor issue here with the "the`.`" part, there should be a whitespace.

Reply:
Also, there is an extra whitespace towards the end.

".... `residents.csv` is  just `residents`."